src/api/Container.js

```diff
@@ -109,7 +124,8 @@ class Container extends React.Component<ContainerProps, ContainerState> {
   startListeningToValue() {
       if (Platform.OS === 'web' && this.animatedValue) {
           this.valueListener = this.animatedValue.addListener(({ value }) => {
-              this.setState({ value });
+              const val = Platform.OS === 'web' ? Math.min(value, this.maxOffset) : value;
```

Comment:
```suggestion
              const val = Math.min(value, this.maxOffset);
```
It's already 'web' as per the check above